pkg/destroy/aws/aws.go

```diff
@@ -868,8 +868,8 @@ func terminateEC2Instance(ctx context.Context, ec2Client *ec2.EC2, iamClient *ia
 }
 
 func terminateEC2InstanceByInstance(ctx context.Context, ec2Client *ec2.EC2, iamClient *iam.IAM, instance *ec2.Instance, logger logrus.FieldLogger) error {
-	// Skip 'shutting-down' and 'terminated' instances since they take a while to get cleaned up
-	if instance.State == nil || *instance.State.Name == "shutting-down" || *instance.State.Name == "terminated" {
+	// Skip 'terminated' instances since they take a while to get cleaned up
+	if instance.State == nil || *instance.State.Name == "terminated" {
```

Comment:
This will re-send the terminate request an instance every 10 seconds while it is shutting down. Can we figure out some way to throttle the terminate requests?